### ISSUE TITLE
opt-out clarity and google analytics on cookie refuse

### ISSUE THREAD

**Issue by roiLeo** [kodadot/nft-gallery#9152]
remove GA & Clarity integration if user click on decline Cookies

**Comment by exezbcz** [comment]
and if they dont accept or decline?

**Comment by roiLeo** [comment]
> and if they dont accept or decline?

I don't think it's possible to hide cookie banner (no close button)
untill user don't specify "accept" we shouldn't session

**Comment by yangwao** [comment]
> remove GA & Clarity integration if user click on decline Cookies

Yes, we discussed this before as we need each datapoint so I would opt for collecting all possible cases for now even before click, so banner can state "Being on our site, you agree with our cookie policy"
So if people can click on decline and redirect them to https://en.wikipedia.org/wiki/Art
